apps/microtvm/reference-vm/README.md

```diff
@@ -15,53 +15,86 @@
-*** `Vagrantfile.packer-template` - Packer template Vagrantfile which will be used to build the
-    base box.
-*** `test-config.json` - JSON file explaining how to perform release tests to `base-box-tool.py`
 
 ## Creating Releases
 
-1. Build the base box for the given platform: `$ ./base-box-tool.py [--provider=<provider>] build <platform>`
-2. Run release tests for each platform:
-    1. Connect any needed hardware to the VM host machine.
-    2. Run tests: `$ ./base-box-tool.py [--provider=<provider>] test [--microtvm-platform=<platform>] <platform> [--test-device-serial=<serial>]`. This
-       command does the following for each provider:
-        1. Copies all files inside `./<platform>` except `.vagrant` and `base-box` to
-           `./release-test`. This is done to avoid reusing any VM the developer may have started.
-        2. Executes `$ vagrant up [--provider=<provider>]`.
-        3. Finds an attached USB device matching the VID and PID specified in `test-config.json`,
-           and if `--test-device-serial` was given, that serial number (as reported to USB). Creates
-           a rule to autoconnect this device to the VM, and also attaches it to the VM>
-        4. SSHs to the VM, `cd` to the TVM root directory, and runs `test_cmd` from
-           `test-config.json`. Nonzero status means failure.
-3. If release tests fail, fix them and restart from step 1.
-4. If release tests pass: `$ ./base-box-tool.py [--provider=<provider>] release <--release-version=<version>> <--platform-version=<version>> <platform>`. Be sure you've logged
-   in to Vagrant Cloud using the `vagrant` tool.
+1. **Build** the base box for a given platform:
+```bash
+$ ./base-box-tool.py [--provider=<provider>] build <platform>
+```
+
+2. **Run** release tests for each platform:
+
+   A. Connect any needed hardware to the VM host machine;
+
+   B. Run tests:
+   ```bash
+   $ ./base-box-tool.py [--provider=<provider>] test [--microtvm-platform=<platform>] [--test-device-serial=<serial>] <platform>
+   ```
+
+   This command does the following for the specified provider:
+
+   * Copies all files inside `./<platform>` except `.vagrant` and `base-box` to
+   `./release-test`. This is done to avoid reusing any VM the developer may have
+   started;
+
+   * Executes `$ vagrant up [--provider=<provider>]`;
+
+   * Finds an attached USB device matching the VID and PID specified in
+   `test-config.json`, and if `--test-device-serial` was given, that serial
+   number (as reported to USB). Creates a rule to autoconnect this device to the
+   VM, and also attaches it to the VM;
+
+   * SSHs to the VM, `cd` to the TVM root directory, and runs `test_cmd` from
+   `test-config.json`. Nonzero status means failure.
+
+3. If release tests _fail_, fix them and restart from step 1.
+
+4. If release tests pass, **release** the box:
+```bash
+$ ./base-box-tool.py [--provider=<provider>] release <--release-version=<version>> <--platform-version=<version>> <platform>
```

Comment:
There is some ambiguity in syntax here around <> vs. []. Is that intentional to represent mandatory vs optional parameters? One example also would be useful, I think.

```suggestion
$ ./base-box-tool.py [--provider=PROVIDER] [--release-version=RELEASE_VER] [--platform-version=PLATFORM_VER] release PLATFORM
```

Reply:
Yeah I agree. So, since `--release-version` and `platform-version` are mandatory I removed `<>`. PTAL on v2. Hopefully it's a bit better now.

Reply:
I also fixed other parts of the documentation replacing <platform> by new PLATFORM. I also expanded a bit more in the dir organization diagram about what's the relation of the PLATFORM/ directories and the PLATFORM parameter passed to base-box-tool.py commands.